### PR TITLE
mention integrity of policies, Endorsements, Reference Values

### LINKED ISSUE #378: section 12 seems silent on
-- The overall Section 12 seems silent on:

o Endorsers and endorsements?
o Reference values?
o What is the implication of combining roles into a single entity as described in 
Section 3.4 and 6.  Does this lack of separation present any additional issues?
o Compositional devices per Section 3.3?

### PR DESCRIPTION
**Pull request by mcr**
close #378

### CODE REVIEW

**Comment by mcr** [comment]
close #383

**Comment by mcr** [comment]
May be duplicated by dthaler pull request #391, second part might belong in privacy considerations instead of here.
@mcr to redraft.

**Review by nedmsmith — APPROVED** [review]

File: draft-ietf-rats-architecture.md [hunk 1]
```diff
@@ -1286,14 +1286,19 @@ For a discussion on the security of epoch IDs see {{epochids-sec}}.
...
+
 This information might be particularly interesting to many attackers.
 For example, knowing that a device is
 running a weak version of firmware provides a way to aim attacks better.
 
-Many Claims in Evidence, many Claims in Attestation Results, and Appraisal Policies potentially contain
+In some circumstances, if an attacker can become aware of appraisal policies, it could potentially provide an attacker with insight into defensive mitigations.
+It is recommended that attention be paid to confidentiality of policies.
+
+Additionally, many Claims in Evidence, many Claims in Attestation Results, and Appraisal Policies potentially contain
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:1301]: The same (both line 1298 and line 1301) can be said of Endorsements and Reference Values.   That's why you modified line 1423 but the same applies here too I think.

Comment (mcr) [reply draft-ietf-rats-architecture.md]: I pushed another commit to update this. I meant to do it as a Suggestion, but github didn't let me.
(I eventually realized that I had gotten signed out!)
Please review again. Oxford comma included.

File: draft-ietf-rats-architecture.md [hunk 2]
```diff
@@ -1286,14 +1286,19 @@ For a discussion on the security of epoch IDs see {{epochids-sec}}.
...
 to provide reliable information about the type of the device and the
 firmware/software that the device is running.
+
 This information might be particularly interesting to many attackers.
 For example, knowing that a device is
 running a weak version of firmware provides a way to aim attacks better.
 
-Many Claims in Evidence, many Claims in Attestation Results, and Appraisal Policies potentially contain
+In some circumstances, if an attacker can become aware of Endorsements, Reference Values, or appraisal policies, it could potentially provide an attacker with insight into defensive mitigations.
+It is recommended that attention be paid to confidentiality of policies.
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:1299]:
```suggestion
It is recommended that attention be paid to confidentiality of such information.
```
(not just policies, but everything list in previous sentence)

File: draft-ietf-rats-architecture.md [hunk 3]
```diff
@@ -1286,14 +1286,19 @@ For a discussion on the security of epoch IDs see {{epochids-sec}}.
...
+
+Additionally, many Claims in Evidence, many Claims in Attestation Results, and Appraisal Policies potentially contain
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:1301]: Appraisal Policies is capitalized here but lower case in 1298.  Be consistent.

File: draft-ietf-rats-architecture.md [hunk 4]
```diff
@@ -1286,14 +1286,19 @@ For a discussion on the security of epoch IDs see {{epochids-sec}}.
...
+
 This information might be particularly interesting to many attackers.
 For example, knowing that a device is
 running a weak version of firmware provides a way to aim attacks better.
 
-Many Claims in Evidence, many Claims in Attestation Results, and Appraisal Policies potentially contain
+In some circumstances, if an attacker can become aware of Endorsements, Reference Values, or appraisal policies, it could potentially provide an attacker with insight into defensive mitigations.
+It is recommended that attention be paid to confidentiality of such information.
+
+Additionally, many Claims in Evidence, many Claims in Attestation Results, and Appraisal Policies potentially contain
```
Comment (mcr) [inline draft-ietf-rats-architecture.md:1301]:
```suggestion
Additionally, many Claims in Evidence, many Claims in Attestation Results, and appraisal policies potentially contain
```

Comment (mcr) [reply draft-ietf-rats-architecture.md:1301]: I think it should be lower case everywhere.

**Review by William-PanWei — APPROVED** [review]

**Pull request merged**